app/V1Module/presenters/GroupsPresenter.php

```diff
@@ -512,6 +548,17 @@ public function actionAddStudent(string $id, string $userId) {
     if ($group->isOrganizational()) {
       throw new InvalidArgumentException("It is forbidden to add students to organizational groups");
```

Comment:
Again... not really sure if this should be in authorization check

Reply:
I'm not certain either. However, I think that if the API call is likely to end with an InvalidArgumentException, we should not give the user false hopes in the check.

Reply:
Well not convinced in either ways... independent opinion by @krulis-martin ?

Reply:
In my opinion, the security endpoint check should follow the behavior of regular endpoint as closely as possible. I.e., if it is forbidden to add students to organizational group, then security should reflect this in authorization.
(Despite the name "security endpoint", I think the actual function is kind of preflight validation.)

Reply:
Ok, that is the thing, I am thinking of it as really just a security endpoint. But if it seems ok to you...